requirements.d/development.txt

```diff
@@ -2,4 +2,5 @@ tox
 mock
 pytest
 pytest-cov<2.0.0
+-e git://github.com/ionelmc/pytest-benchmark.git#egg=pytest-benchmark
```

Comment:
need to check again, if there is a5 or release, I maybe don't need to pull from git master.